test/test_shape_ops.py

```diff
@@ -386,11 +386,11 @@ def test_flip(self, device):
         self.assertEqual(flip1_result, data.flip(1))
 
         # test empty tensor, should just return an empty tensor of the same shape
-        data = torch.tensor([])
+        data = torch.tensor([]).to(device)
         self.assertEqual(data, data.flip(0))
 
         # test bool tensor
-        a = torch.tensor([False, True])
+        a = torch.tensor([False, True]).to(device)
```

Comment:
Also a good fix for the existing test, but like the above the device can be passed to `torch.tensor()`.